composer.json

```diff
@@ -39,10 +39,10 @@
         }
     },
     "require": {
-        "php": ">=7.4",
+        "ext-protobuf": "*",
         "ext-grpc": "*",
+        "php": ">=7.4",
         "firebase/php-jwt": "^6.3",
-        "google/protobuf": "3.22.3",
         "grpc/grpc": "1.52.0",
```

Comment:
is it possible to bump this to 1.57.0?

Reply:
Yea good call, I'll open a new pr with that change and make sure it gets into our next release

Reply:
https://github.com/momentohq/client-sdk-php/pull/180